source/Ship.cpp

```diff
@@ -272,6 +272,8 @@ void Ship::Load(const DataNode &node)
 			displayModelName = child.Token(1);
 		else if(key == "plural" && child.Size() >= 2)
 			pluralModelName = child.Token(1);
+		else if(key == "variant map shipyard name" && child.Size() >= 2)
```

Comment:
```suggestion
		else if(key == "variant shipyards map name" && child.Size() >= 2)
```
doesn't really matter, but looks better imo

Reply:
I think it's better the current way